.pre-commit-config.yaml

```diff
@@ -17,3 +17,10 @@ repos:
           - --autofix
       - id: end-of-file-fixer
       - id: trailing-whitespace
+  - repo: local
+    hooks:
+      - id: yaml-extension
+        name: yaml-extension
```

Comment:
Looks like this "name" overrides the default one, probably should leave it off.

Reply:
Actually. I'll change it to match the others with sentence case